source/utils/moneyFormatter.ts

```diff
@@ -1,10 +1,16 @@
-const formatDuffs = (
-  duffsAmount: number,
-  decimalPlaces = 8,
+const formatDash = (duffsAmount: number, decimalPlaces = 8): number => {
+  return duffsAmount / 10 ** decimalPlaces;
+};
+
+const formatMoneyToString = (
```

Comment:
Could be just `formatMoney`. `string` is implied by specifying it in return argument type